src/sentry/models/organizationmembermapping.py

```diff
@@ -44,6 +47,9 @@ class OrganizationMemberMapping(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_organizationmembermapping"
-        unique_together = (("organization_id", "user"), ("organization_id", "email"))
+        unique_together = (
+            ("organization_id", "organizationmember_id", "user"),
+            ("organization_id", "organizationmember_id", "email"),
+        )
```

Comment:
Are you sure you want to remove the old constraints? Since `organizationmember_id` is null for now, the constraints won't apply at all, so you could end up with duplicate rows.

Does it make sense to remove the constraints after we backfill?

Reply:
:+1: to keeping the existing constraints around until we can backfill the new column.